AxonDeepSeg/segment.py

```diff
@@ -260,8 +253,6 @@ def main(argv=None):
     with open("axondeepseg.log", "a") as f:
         f.write("===================================================================================\n")
 
-    logger.info(AxonDeepSeg.__version_string__)
```

Comment:
Would it still be useful to have the version logged in another way? To be fair, I don't think so because we would always encourage users to use the master branch

Reply:
Yes I agree - I think we should open a new issue to handle this and find an alternative solution, I'll sdo that now